experimental/packages/opentelemetry-exporter-metrics-otlp-grpc/README.md

```diff
@@ -83,10 +90,17 @@ counter.add(10, { 'key': 'value' });
 Apache 2.0 - See [LICENSE][license-url] for more information.
 
 [discussions-url]: https://github.com/open-telemetry/opentelemetry-js/discussions
+
```

Comment:
Do we need to add these blank lines? I didn't find it to be the common pattern.

Reply:
Looks like WebStorm auto-format added those :sweat_smile: Removed in [1f66dc9](https://github.com/open-telemetry/opentelemetry-js/pull/3305/commits/1f66dc9b8aa392ffe3a486b62278b6c8463abb32)